dags/annual_reports/utils.py

```
@@ -0,0 +1,38 @@
+import os
+import xml.etree.ElementTree as ET
+
+from .constants import ONLY_CERN_SUBJECTS
+
+
+def get_endpoint(key, year):
+    cds_token = os.environ.get("CDS_TOKEN", "")
+    if cds_token:
```

the token should be always be there, we can raise and exception if it's missing